azkaban-common/src/main/java/azkaban/scheduler/TriggerBasedScheduleLoader.java

```diff
@@ -43,10 +44,10 @@
 
   private long lastUpdateTime = -1;
 
-  public TriggerBasedScheduleLoader(final TriggerManager triggerManager,
-      final String triggerSource) {
+  @Inject
+  public TriggerBasedScheduleLoader(final TriggerManager triggerManager) {
     this.triggerManager = triggerManager;
-    this.triggerSource = triggerSource;
+    this.triggerSource = ScheduleManager.triggerSource;
```

Comment:
How does Guice know to inject ScheduleManager here if it isn't a parameter of TriggerBasedScheduleLoader?